### DISCUSSION TITLE
[Feature Request] Ability to duplicate an existing user group

### DISCUSSION THREAD

**Discussion by Ticolyle** [craftcms/cms#11795] [Ideas] [answered]
When setting up new users groups it would be convenient to be able to replicate an existing group. This could help save time when selecting the same options multiple times for many groups.

**Comment by brandonkelly** [comment, marked as answer]
Agree it would be helpful. In the meantime, if you don’t mind getting your hands a little dirty, user groups are one of the easier things to duplicate by copying the project config data:

1. Create a new UUID for the new group from https://www.uuidgenerator.net/
2. Find the existing group in `config/project/users/groups/`, and duplicate it named after your new group handle + UUID (e.g. `myNewHandle-xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx.yaml`)
3. Edit the settings within the file (at the very least give it a new `name` and `handle`
4. Run `php craft project-config/apply`
